test/functional/ui/linematch_spec.lua

```diff
@@ -1084,6 +1151,436 @@ something
+       {3:Xtest-functional-diff-screen-1.2                  }{2:Xtest-functional-diff-screen-1                    }|
+       :set diffopt+=chardiff:100                                                                          |
+        ]])
+      end)
+    end)
+  end)
+  describe('show a diff with charmatch enabled, with same UTF-8 character', function()
+    before_each(function()
+      local f1 = [[
+aaaहaaa
+      ]]
+      local f2 = [[
+aaaहaaa
+      ]]
+      write_file(fname, f1, false)
+      write_file(fname_2, f2, false)
+      reread()
+    end)
+    describe('normal comparison, including whitespace', function()
+      before_each(function()
+        feed(':set diffopt+=chardiff:100<cr>')
+      end)
+      it('display results', function()
+        screen:expect([[
+       {7:+ }{8:  1 }{13:^+--  2 lines: aaaहaaa······················}│{7:+ }{8:  1 }{13:+--  2 lines: aaaहaaa·······················}|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
+       {1:~                                                }│{1:~                                                 }|
```

Comment:
screen.expect now supports a feature that avoids redundant lines in the tests. it looks like this:

```suggestion
       {1:~                                                }│{1:~                                                 }|*14
```

please use this where possible.

Reply:
all the duplicate lines are now using the *n